.gitmodules

```diff
@@ -5,3 +5,6 @@
 	path = cmake/cable
 	url = https://github.com/ethereum/cable
 	branch = master
+[submodule "ethash"]
```

Comment:
since i don't like submodules and also since we have cmake/hunter i would suggest make either ethash module a hunter package or use cmakes external project directly for that dependency.

For cable i understand the submodule approach but for this?

Reply:
for cable subtree is probably better than submodule.